src/app/modules/shared/models/constraints/concept-constraint.ts

```diff
@@ -1,12 +1,17 @@
 import {Constraint} from './constraint';
 import {Concept} from "../concept";
 import {Value} from "../value";
+import {DateOperatorState} from "../../../data-selection/constraint-components/concept-constraint/date-operator-state";
 
 export class ConceptConstraint implements Constraint {
 
   private _type: string;
   private _concept:Concept;
   private _values: Value[];
+  applyDateConstraint: boolean = false;
+  dateOperator: DateOperatorState = DateOperatorState.BETWEEN;
+  date1: Date = new Date();
+  date2: Date = new Date();
```

Comment:
Not necessarily, as they can be either the limits of a range or a reference date, depending on the operator. I also wanted to avoid confusion with the observation's start and end date.